lametro/templatetags/lametro_extras.py

```diff
@@ -125,10 +125,11 @@ def get_minutes(event_id):
         sqs = SearchQuerySet().filter(content=content).all()
         if sqs:
             for q in sqs:
-                if q.object.bill_type == 'Minutes' and q.object.slug:
-
-                    if re.search(content, q.object.ocr_full_text, re.IGNORECASE):
-                        return '/board-report/' + q.object.slug
+                if (q.object.bill_type == 'Minutes' and 
+                    q.object.slug and 
+                    q.object.ocr_full_text):
+                        if re.search(content, q.object.ocr_full_text, re.IGNORECASE):
```

Comment:
@fgregg I do not see an indentation problem, when I expand the code and place this snippet in context. Can you say more about what you expect?